docker/gha-runner.Dockerfile

```diff
@@ -44,3 +44,12 @@ RUN curl \
     | bash
 RUN cargo binstall --no-confirm --no-symlinks cargo-risczero
 RUN cargo risczero install
+
+# Install Zombienet and copy Polkadot binaries, which are all required for xtask tests
+RUN curl -sL https://deb.nodesource.com/setup_20.x | bash -
+RUN apt-get install -y nodejs multitail
+RUN npm install -g @zombienet/cli
+
+COPY --from=parity/polkadot:v1.6.0 /usr/bin/polkadot /usr/bin/
```

Comment:
I discovered that using ARG is more complex than expected. Doing something like this:

```
ARG POLKADOT=parity/polkadot:v1.6.0
FROM $POLKADOT as polkadot
COPY --from=polkadot /usr/bin/polkadot /usr/bin/
```

does not work because ARG can be defined at build time with an argument variable, and `parity/polkadot:v1.6.0` would be used only if it is not specified. However, if it is specified but not assigned to anything, it would be empty, which is not valid for the FROM command. To make it work, it should be:

```
ARG POLKADOT=parity/polkadot:v1.6.0
FROM ${POLKADOT:-parity/polkadot:v1.6.0} as polkadot
COPY --from=polkadot /usr/bin/polkadot /usr/bin/
```

But if the goal is to reduce the lines that need to be changed on a Polkadot update, I opted for:

```
FROM parity/polkadot:v1.6.0 as polkadot
COPY --from=polkadot /usr/bin/polkadot /usr/bin/
```